src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  justify-content: space-between;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__top {
+    width: 160px;
+    height: 134px;
+    display: block;
+    padding-top: 16px;
+    margin-bottom: 40px;
+    align-self: center;
+  }
+
+  &__title {
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    align-self: start;
+    padding-top: 4px;
+    margin: 0;
+  }
+
+  &__rates {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+    padding-bottom: 12px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
```

Comment:
@Helen-arch Here you use `margin-bottom` but in `rates` you use `margin-top`. Try to use only one way of margins top or bottom in your project to avoid margin collapse https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_box_model/Mastering_margin_collapsing